File: app/models/product.rb
# coding: utf-8

class Product < ActiveRecord::Base
  attr_accessible :asin, :category, :creater_name, :image_url, :item_attributes, :item_url, :publisher, :release_date, :title

  validates :asin, :presence => true
  validates :category, :presence => true
  validates :creater_name, :presence => true
  validates :item_url, :presence => true
  validates :title, :presence => true
  validates :release_date, :presence => true
end

File: app/assets/javascripts/models/product.js
define(['backbone'], function() {
  var Product = Backbone.Model.extend({
    default: function() {
      return {
        'asin': '',
        'title': '',
        'category': '',
        'creater_name': '',
        'image_url': '',
        'item_url': ''
      };
    }
  });
  return Product;
});
